workspaces/liferay-customer-workspace/client-extensions/liferay-customer-custom-element/src/routes/security-vulnerabilities/pages/SecurityVulnerabilitiesItem/SecurityVulnerabilitiesItem.tsx

```diff
@@ -212,7 +212,7 @@ const SecurityVulnerabilitiesItem = () => {
 						{jiraIssue[JiraEnum.FIELDS]?.[JiraEnum.CATEGORIES] && (
 							<div className="mb-4">
 								<h5 className="text-neutral-10">
-									{i18n.translate('categories')}
+									{i18n.translate('category')}
```

Comment:
are we sure we want these singular? feels odd when there are multiple listed right below it. If we do want it singular, should we also change Fix Versions to Fix Version?

Reply:
I think they should match the singular that Tiffany wanted on the search page. So I think this makes sense.

Reply:
okay, so then also update Fix Versions to be singular?

Reply:
yeah, lets go singular with that one also, to stay consistent.

Reply:
Done!